service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/FileUploadNotificationReceiver.java

```diff
@@ -32,7 +34,13 @@
      * @param sslContext the SSL context to use during the TLS handshake when opening the connection. If null, a default
      *                   SSL context will be generated. This default SSLContext trusts the IoT Hub public certificates.
      */
-    FileUploadNotificationReceiver(String hostName, String userName, String sasToken, IotHubServiceClientProtocol iotHubServiceClientProtocol, ProxyOptions proxyOptions, SSLContext sslContext)
+    FileUploadNotificationReceiver(
+            String hostName,
+            String userName,
```

Comment:
q - what is the difference between hostName and userName?

Reply:
userName in this context refers to the shared access key name, whereas hostname is "my-iot-hub.azure-devices.net"

Since we are moving away from connection strings, these username constructors will largely be unused